packages/headless/src/insight.index.ts

```diff
@@ -33,3 +33,8 @@ export {buildController} from './controllers/controller/headless-controller';
 export type {InsightInterfaceState} from './features/insight-interface/insight-interface-state';
 export type {InsightInterface} from './controllers/insight-interface/insight-interface';
 export {buildInsightInterface} from './controllers/insight-interface/insight-interface';
+export type {
```

Comment:
@lbergeron @aelfaleh hey guys, just tagging you here to confirm that this is the right thing to do

Reply:
So the plan on our side is to export the insight controllers under the same name as their search page equivalent. The aim is for the Quantic UI components to reference the same build function using the bundle that has been loaded. This is to maximise code reuse and avoid having to create another set of quantic components.
What are your thoughts on that?

Reply:
So it's confirmed that we'll be exporting all the insight controllers under the same names as their search counterparts. The plan is to rename them to the correct name in their respective controllers/ files. That process is currently underway in a few PRs.

Reply:
no problem, we can have a PR that changes all of this later!